models/issues/issue_project.go

```diff
@@ -140,10 +144,6 @@ func addUpdateIssueProject(ctx context.Context, issue *Issue, doer *user_model.U
 		return err
 	}
 
-	if err := issue.LoadRepo(ctx); err != nil {
```

Comment:
CreateComment does need that info ... (L151)

Reply:
This has been moved to line 128 because line 138 need that, it's not removed.